packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -735,7 +735,8 @@ class _ModalBottomSheetState<T> extends State<_ModalBottomSheet<T>> {
 ///    and then becomes scrollable once it reaches its maximum size.
 ///  * [DisplayFeatureSubScreen], which documents the specifics of how
 ///    [DisplayFeature]s can split the screen into sub-screens.
-///  * <https://material.io/design/components/sheets-bottom.html#modal-bottom-sheet>
+///  * The Material 2 spec at <https://m2.material.io/components/sheets-bottom>.
+///  * The Material 3 spec at <https://m3.material.io/components/bottom-sheets/overview>.
 class ModalBottomSheetRoute<T> extends PopupRoute<T> {
```

Comment:
Same update would probably be good in the other docs that refer to the old link:

```
$ git grep --heading -A1 material.io/design/components/sheets-bottom
packages/flutter/lib/src/material/bottom_sheet.dart
///  * <https://material.io/design/components/sheets-bottom.html>
class BottomSheet extends StatefulWidget {
--
///  * <https://material.io/design/components/sheets-bottom.html#modal-bottom-sheet>
Future<T?> showModalBottomSheet<T>({
--
///  * <https://material.io/design/components/sheets-bottom.html#standard-bottom-sheet>
PersistentBottomSheetController<T> showBottomSheet<T>({
--
packages/flutter/lib/src/material/scaffold.dart
  ///  * <https://material.io/design/components/sheets-bottom.html#standard-bottom-sheet>
  PersistentBottomSheetController<T> showBottomSheet<T>(
```

Reply:
Thanks! One of these remains, and I think it would be good there too:
```
$ git grep --heading -nA1 material.io/design/components/sheets-bottom
packages/flutter/lib/src/material/bottom_sheet.dart
70:///  * <https://material.io/design/components/sheets-bottom.html>
71-class BottomSheet extends StatefulWidget {
```